playwright/tests/inscription/mock/browser/conditionBuilder.spec.ts

```diff
@@ -0,0 +1,61 @@
+import type { Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { InscriptionView } from '../../../page-objects/inscription/InscriptionView';
+import { browserBtn } from './browser-mock-utils';
+
+test.describe('Condition Builder', () => {
```

Comment:
IMO it is not needed to add a describe block, this only makes the path of the test longer.
I think this makes only sense if you want to group some tests together and it makes no sense to split them into multiple test files.